backend/README.md

```
@@ -0,0 +1,125 @@
+
+### Prerequisites
+- Go 1.21 or later (install from https://golang.org/dl/)
+
+### Run the first server:
+```bash
+cd backend
+go run main.go
+```
+
+Expected output:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)